webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -2088,13 +2134,17 @@ class StringFormatter private (
    *                    user IRI.
    * @return the same optional string.
    */
+  @deprecated("Use validateAndEscapeOptionalUserIri(Option[String]) instead.")
   def validateAndEscapeOptionalUserIri(
     maybeString: Option[String],
     errorFun: => Nothing
   ): Option[String] = // V2 / value objects
+    validateAndEscapeOptionalUserIri(maybeString).getOrElse(errorFun)
+
+  def validateAndEscapeOptionalUserIri(maybeString: Option[String]): Validation[ValidationException, Option[String]] =
```

Comment:
Consider removing this method and inline the original `validateAndEscapeOptionalUserIri` instead.